packages/compartment-mapper/src/parse-json.js

```diff
@@ -2,19 +2,24 @@
 
 import { parseLocatedJson } from './json.js';
 
-const { freeze } = Object;
+/**
+ * TypeScript cannot be relied upon to deal with the nuances of Readonly, so we
+ * borrow the pass-through type definition of harden here.
+ *
+ * @type {import('ses').Harden}
```

Comment:
Same comment here (maybe we need a `LaxFreeze` in `types.js`)

Reply:
Antifreeze?